rusk-wallet/src/bin/io/prompt.rs

```diff
@@ -98,38 +87,30 @@ where
     S: std::fmt::Display,
 {
     // inform the user about the mnemonic phrase
-    println!("The following phrase is essential for you to regain access to your wallet\nin case you lose access to this computer.");
-    println!("Please print it or write it down and store it somewhere safe:");
-    println!();
-    println!("> {}", phrase);
-    println!();
+    let msg = format!("The following phrase is essential for you to regain access to your wallet\nin case you lose access to this computer. Please print it or write it down and store it somewhere safe.\n> {} \nHave you backed up this phrase?", phrase);
 
     // let the user confirm they have backed up their phrase
-    loop {
-        let q = requestty::Question::confirm("proceed")
-            .message("Have you backed up your mnemonic phrase?")
-            .build();
-
-        let a = requestty::prompt_one(q)?;
-        if a.as_bool().expect("answer to be a bool") {
-            return Ok(());
-        }
+    let confirm = Confirm::new(&msg)
+        .with_help_message(
+            "It is important you backup the mnemonic phrase before proceeding",
+        )
+        .prompt()?;
+
+    if !confirm {
+        confirm_mnemonic_phrase(phrase)?
     }
+
+    Ok(())
 }
 
 /// Request the user to input the mnemonic phrase
 pub(crate) fn request_mnemonic_phrase() -> anyhow::Result<String> {
     // let the user input the mnemonic phrase
     let mut attempt = 1;
     loop {
-        let q = Question::input("phrase")
-            .message("Please enter the mnemonic phrase:")
-            .build();
+        let phrase = Text::new("Please enter the mnemonic phrase").prompt()?;
```

Comment:
```suggestion
        let phrase = Text::new("Please enter the mnemonic phrase:").prompt()?;
```
Same here. First character is capitalized, unlike how it's done in other places, and the `:` is missing.